traffic_ops/traffic_ops_golang/dbhelpers/db_helpers_test.go

```diff
@@ -408,3 +408,55 @@ func TestGetCDNIDFromName(t *testing.T) {
 	}
 
 }
+
+func TestGetSCInfo(t *testing.T) {
+	var testCases = []struct {
+		description   string
+		name          string
+		expectedError error
+		exists        bool
+	}{
+		//{
+		//	description:   "Success: Get valid SC",
+		//	name:          "hdd",
+		//	expectedError: nil,
+		//	exists:        true,
+		//},
+		{
+			description:   "Failure: SC not in DB",
+			name:          "disk",
+			expectedError: errors.New("error getting server capability info: sql: no rows in result set"),
+			exists:        false,
+		},
+	}
+	for _, testCase := range testCases {
+		t.Run(testCase.description, func(t *testing.T) {
+			t.Log("Starting test scenario: ", testCase.description)
+			mockDB, mock, err := sqlmock.New()
+			if err != nil {
+				t.Fatalf("an error '%s' was not expected when opening a stub database connection", err)
+			}
+			defer mockDB.Close()
+
+			db := sqlx.NewDb(mockDB, "sqlmock")
+			defer db.Close()
+
+			rows := sqlmock.NewRows([]string{"name"})
+			mock.ExpectBegin()
+			if testCase.exists {
+				rows = rows.AddRow(testCase.name)
+			}
+			mock.ExpectQuery("SELECT").WillReturnRows(rows)
+			mock.ExpectCommit()
+
+			scExists, err := GetSCInfo(db.MustBegin().Tx, testCase.name)
+			if testCase.exists != scExists {
+				t.Errorf("Expected return exists: %t, actual %t", testCase.exists, scExists)
+			}
+
+			if err == testCase.expectedError {
+				t.Errorf("getSCInfo expected: %s, actual: %s", testCase.expectedError, err)
+			}
+		})
+	}
+}
```

Comment:
So, it seems you've commented out the positive test, and I think I know why. I was puzzled how the negative test would be passing since two `error`s aren't equal just because they have the same message, but then I realized: your negative test passing is a false positive, because your test fails *if the error you get is equal to the error you expect*, which is the opposite of what expectation implies. Line 457 should be doing the opposite of what it's doing, but that alone won't pass, because you also need the correct error specified on line 428 - I believe you're looking for `sql.ErrNoRows`, and that comparison should be done with `errors.Is`.